src/main/java/de/dennisguse/opentracks/ui/aggregatedStatistics/dailyStats/DailyRunsProvider.java

```diff
@@ -1,8 +1,201 @@
+        runs.add(dummyRun5);
+        runs.add(dummyRun6);
+        runs.add(dummyRun7);
+        runs.add(dummyRun8);
+        runs.add(dummyRun9);
+        runs.add(dummyRun10);
+        return runs;
+    }
+    private static List<RunVM> getDailyRunsSample5()
+    {
+        ArrayList<RunVM> runs = new ArrayList<RunVM>();
+
+        RunVM dummyRun1 = new RunVM(runIdCtr, 12.5f, 0.350f, 6.5f, 900);
+        runIdCtr++;
+        RunVM dummyRun2 = new RunVM(runIdCtr, 14.1f, 0.44f, 5.75f, 1100);
+        runIdCtr++;
+        RunVM dummyRun3 = new RunVM(runIdCtr, 11.7f, 0.33f, 5.5f, 1200);
+        runIdCtr++;
+        RunVM dummyRun4 = new RunVM(runIdCtr, 13.8f, 0.385f, 5.25f, 1000);
+        runIdCtr++;
+        RunVM dummyRun5 = new RunVM(runIdCtr, 13.9f, 0.403f, 5.15f, 950);
+        runIdCtr++;
+        RunVM dummyRun6 = new RunVM(runIdCtr, 12.1f, 0.38f, 4.85f, 975);
+        runIdCtr++;
+        RunVM dummyRun7 = new RunVM(runIdCtr, 7.1f, 0.21f, 3.5f, 6000);
+        runIdCtr++;
+        RunVM dummyRun8 = new RunVM(runIdCtr, 6.667f, 0.18f, 3.8f, 4500);
+        runIdCtr++;
+        RunVM dummyRun9 = new RunVM(runIdCtr, 6.9f, 0.23f, 5.2f, 5000);
+        runIdCtr++;
+        RunVM dummyRun10 = new RunVM(runIdCtr, 3.3f, 0.133f, 5.4f, 7500);
+        runIdCtr++;
+        RunVM dummyRun11 = new RunVM(runIdCtr, 3f, 0.105f, 5.8f, 12000);
+        runIdCtr++;
+        RunVM dummyRun12 = new RunVM(runIdCtr, 3.55f, 0.155f, 2.9f, 8000);
+        runIdCtr++;
+        RunVM dummyRun13 = new RunVM(runIdCtr, 13.4f, 0.375f, 4.2f, 1050);
+        runIdCtr++;
+        RunVM dummyRun14 = new RunVM(runIdCtr, 14.1f, 0.388f, 3.9f, 1025);
+        runIdCtr++;
+
+        runs.add(dummyRun1);
+        runs.add(dummyRun2);
+        runs.add(dummyRun3);
+        runs.add(dummyRun4);
+        runs.add(dummyRun5);
+        runs.add(dummyRun6);
+        runs.add(dummyRun7);
+        runs.add(dummyRun8);
+        runs.add(dummyRun9);
+        runs.add(dummyRun10);
+        runs.add(dummyRun11);
+        runs.add(dummyRun12);
+        runs.add(dummyRun13);
+        runs.add(dummyRun14);
+        return runs;
+    }
+
+    public static List<RunVM> getAllDailyRuns()
+    {
+        int randomNum = (int) (Math.random()*10)+1;
+
+        if(randomNum == 1 || randomNum == 2)
+        {
+            return getDailyRunsSample1();
+        }
+        else if(randomNum == 3 || randomNum == 4)
+        {
+            return getDailyRunsSample2();
+        }
+        else if(randomNum == 5 || randomNum == 6)
+        {
+            return getDailyRunsSample3();
+        }
+        else if(randomNum == 7 || randomNum == 8)
+        {
+            return getDailyRunsSample4();
+        }
+        else if(randomNum == 9 || randomNum == 10)
+        {
+            return getDailyRunsSample5();
+        }
+        else
+        {
+            //This shouldn't ever be called since randomNum will generate a number from 1-10 inclusive.
+            return new ArrayList<RunVM>();
+        }
```

Comment:
Since we know the only possible options are 9 and 10 here, you could simply wrap this into an else block and return the 5th sample